counter-script/project/Forc.toml

```diff
@@ -0,0 +1,2 @@
+[workspace]
+members = ["./contracts/counter", "./scripts/interaction_script"]
```

Comment:
This should build the interface but idk if it formats it. Can you change the interface and run the formatter to see if it formats? If nothing changes then add the interface here too

Reply:
It formats